### PR TITLE
[Docker] Update ANTs to 2.2.0

### PR DESCRIPTION
**Pull request by oesteban**
Contributions in this PR:

- Replaces ANTs 2.1.0 with 2.2.0 in the nipype/base docker image

- Depends on #2022

### CODE REVIEW

**Comment by chrisfilo** [comment]
What's the source of the binaries?

**Comment by oesteban** [comment]
https://github.com/kaczmarj/ANTs-builds

File: docker/base.Dockerfile [hunk 1]
```diff
@@ -122,8 +122,8 @@ ENV FSLDIR=/usr/share/fsl/5.0 \
 
 # Installing and setting up ANTs
 RUN mkdir -p /opt/ants && \
-    curl -sSL "https://github.com/stnava/ANTs/releases/download/v2.1.0/Linux_Ubuntu14.04.tar.bz2" \
-    | tar -xjC /opt/ants --strip-components 1
+    curl -sSL "https://www.dropbox.com/s/2f4sui1z6lcgyek/ANTs-Linux-centos5_x86_64-v2.2.0-0740f91.tar.gz?dl=1" \
+    | tar -xzC /opt/ants --strip-components 1
```
Comment (satra) [inline docker/base.Dockerfile]: check the command i added in the other PR: this may install ants in `/opt/ants/ants`

Comment (satra) [inline docker/base.Dockerfile]: actually never mind. the `--strip-components` does the necessary bits.

**Comment by oesteban** [comment]
Close in favor of #2024

**Pull request closed without merging**